superset/initialization/__init__.py

```diff
@@ -473,7 +472,11 @@ def configure_data_sources(self) -> None:
         # Registering sources
         module_datasource_map = self.config["DEFAULT_MODULE_DS_MAP"]
         module_datasource_map.update(self.config["ADDITIONAL_MODULE_DS_MAP"])
-        ConnectorRegistry.register_sources(module_datasource_map)
+
+        # todo(hughhhh): fully remove the datasource config register
+        for module_name, class_names in module_datasource_map.items():
+            class_names = [str(s) for s in class_names]
+            __import__(module_name, fromlist=class_names)
```

Comment:
I was thinking that we wouldn't have to init anything to make this work?

Reply:
for some reason the app won't without this code being here, can we put that into another ticket?